.github/workflows/helm.yml

```diff
@@ -34,6 +34,14 @@ jobs:
         with:
           version: v3.14.2 # remember to also update for the second job (release)
 
+      - name: Install Helm Schema Plugin
+        run: |
+          helm plugin install https://github.com/losisin/helm-values-schema-json.git
+
+      - name: Generate values.schema.json
+        run: |
+          helm schema -input deploy/charts/external-secrets/values.yaml -output deploy/charts/external-secrets/values.schema.json
+
```

Comment:
Adding this to here means we will never be sure that the latest version of values.schema.json is commited, as this is produced on CI time only.

Instead, can we add the generation as a `make` command  (e.g. `make helm.schema.update`) and run it on `make check-diff` ? This will enforce any updates to helm charts that change the schema to run the command on PR time.